src/server/game/Spells/SpellMgr.h

```diff
@@ -467,7 +468,9 @@ struct TC_GAME_API SpellArea
     Gender gender;                                          // can be applied only to gender
     uint32 questStartStatus;                                // QuestStatus that quest_start must have in order to keep the spell
     uint32 questEndStatus;                                  // QuestStatus that the quest_end must have in order to keep the spell (if the quest_end's status is different than this, the spell will be dropped)
-    uint8 flags;                                            // if SPELL_AREA_FLAG_AUTOCAST then auto applied at area enter, in other case just allowed to cast || if SPELL_AREA_FLAG_AUTOREMOVE then auto removed inside area (will allways be removed on leaved even without flag)
+    uint8 flags;                                            // if SPELL_AREA_FLAG_AUTOCAST then auto applied at area enter, in other case just allowed to cast
+                                                            // if SPELL_AREA_FLAG_AUTOREMOVE then auto removed inside area (will allways be removed on leaved even without flag)
```

Comment:
Will this be overwritten by the same file in the PR
**Core/Spells: implement SPELL_AREA_FLAG_AUTOAPPLY** #21758 ?
Otherwise, please replace "on leaved" with 'on leaving' here too.

Reply:
Yes. I will correct it when it gets merged